Program.cs

```diff
@@ -10,6 +10,55 @@ internal class Program
     {
         static void Main(string[] args)
         {
+        Console.WriteLine("Введите натуральное число");
+        int n;
+        if (!TryInputNumber(out n))
+                {
+                    Console.ReadKey();
+                    return;
```

Comment:
Вот здесь следует поставить печать сообщения.
```suggestion
        if (!int.TryParse(Console.ReadLine(), out int n))
                {
                    Console.WriteLine("Ошибка ввода");
                    Console.ReadKey();
                    return;
```